Sources/UI/Experiences/Models/Block.swift

```diff
@@ -20,7 +20,7 @@ public protocol Block: AttributeRepresentable, Decodable {
 }
 
 extension Block {
```

Comment:
`AttributeRepresentable` has been removed in this PR. You'll need to update this to match the approach you took in the `Beacon` and `Geofence` classes.